app/assets/stylesheets/variables.scss

```diff
@@ -13,3 +13,7 @@ $footer-top-margin: 20px;
 $footer-mobile-adjustment: 40px;
 
 $text-muted: $color-black-medium; // with gray body bg, need darker muted color
+
+$pagination-color: $color-blackish;
```

Comment:
I thought you were only applying the custom styles to the active page before?

Reply:
You're correct. I hastily copied the wrong group of variables. Hopefully this last update is good.